conans/test/unittests/util/detected_os_test.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# vim: tabstop=8 expandtab shiftwidth=4 softtabstop=4
+
+import unittest
+import platform
+from mock import mock
+from conans.client.conf.detect import detected_os
+from nose.plugins.attrib import attr
+
+
+class OSInfoTest(unittest.TestCase):
```

Comment:
This is testing a single function, wonderful! 💯 ...but move this file to `unittests/clien/conf/detect/` to reflect the layout of the codebase, it will be easier to find in the future.

Reply:
okay
it a bit misleading that detected_os and detected_architecture are in different modules, but let's stick with current layout

Reply:
moved